twitter_analysis/app.py

```diff
@@ -151,7 +152,7 @@ def convertData():
     for i in topics:
         headers = ['Id','Tweet','Language','Created At','Topic','Query']
         tweets = list(collection.find({'topic':i}))
-        f = open(i+'.csv', 'a')
+        f = open('./folder_1/'+i+'.csv', 'w+')
```

Comment:
similar to readme comment - check for existence of these folders, create folder if you need, add errors and exceptions as youre writing this code.